src/cdk/constructs/domain-manager/resource/route53-manager-access-role.ts

```diff
@@ -7,16 +7,18 @@ import {
 import { MiraConfig } from '../../../../config/mira-config'
 import { MiraStack } from '../../../stack'
 
-export class Route53ManagerAccessRoleStack extends MiraStack {
+export const ROUTE53_MANAGER_POLICY_BOUNDARY = 'Route53ManagerPolicyBoundary'
+
+export class Route53ManagerAccessRole extends MiraStack {
   constructor (parent: Construct) {
-    super(parent, Route53ManagerAccessRoleStack.name)
+    super(parent, Route53ManagerAccessRole.name)
     const { hostedZoneId } = MiraConfig.getDomainConfig()
     if (!hostedZoneId) {
       throw new Error('Cannot find hostedZoneId in config.')
     }
     new ManagedPolicy(this, 'permissionBoundaryPolicy',
       {
-        managedPolicyName: MiraConfig.calculateSharedResourceName('Route53ManagerPolicyBoundary'),
+        managedPolicyName: MiraConfig.getBaseStackName('Route53ManagerPolicyBoundary'),
```

Comment:
Similar to above, I think the semantics we are using is a bit confusing.

Reply:
You're right, the same fix as above.